### ISSUE TITLE
pipe descriptives table

### ISSUE THREAD

**Issue by suwolff** [silkyproject/silkyR#344]
I am not sure this is the right place to ask, but I hope you can help me: I would like to know how I can access a part of the descriptives results object ($descriptivesT) when using the piping operator. For example, without piping, I can do this:

results <- jmv::descriptives(dat, vars = vars(mpg, cyl, disp, gear), desc = 'rows')
tibble.results <- as.tibble(results$descriptivesT)
 
To keep it tidy, I would like to do this without specifying an intermediate variable, and use the pipe instead, like:

tibble. results <- jmv::descriptives(dat, vars = vars(mpg, cyl, disp, gear), desc = 'rows') %>%
      as.tibble(.$descriptivesT)

However, this produces an error, and I do not know if and how this is possible. How do I refer correctly to the $descriptivesT part of the object produced by jmv::descriptives? 

Thank you very much in advance!

**Comment by jonathon-love** [comment]
you fancy kids, and your fancy pipe operators ... when i was young, we wrote *for loops*, ... not even *for-each loops*, plain *for-loops*. we had to define an index, we had to increment that index, and we had to text that index. and we did it all by hand!

and we used data frames, and we were happy! we didn't even know what a tibble was!

how about:

```
jmv::descriptives(mtcars, vars = vars(mpg, cyl, disp, gear), desc = 'rows')$descriptivesT %>% as_tibble()
```

cheers
